scripts/ci/lint

```diff
@@ -52,6 +52,19 @@ echo "::group::Checking eol"
 checkerr "$(check-eol-dry-run)"
 echo "::endgroup::"
 
+check-spdx()
+{
+  for file in $(git ls-files -- ":/" ":/!:libopencm3" ":/!:LICENSE" ":/!:fips202/LICENSE" ":/!:mlkem/LICENSE" ":/!:hal/mps2/LICENSE.txt" ":/!:.git*" ":/!:flake.lock"); do
+    if [[ $(grep "SPDX-License-Identifier:" $file | wc -l) == 0 ]]; then
```

Comment:
It can't already be the first line.
It's not a perfect check anyway (because it also matters what license it is).  
It's meant more to make our life a bit easier when people forget.
Please don't put the license headers anywhere else...